### PR TITLE
[Kotlin Migration] TagsListTest

### PR DESCRIPTION
**Pull request by BrayanDSO**
The only space changes made were made by pre-commit Ktlintformat. If necessary, I'll add a `--no-verify`

## Checklist
_Please, go through these checks before submitting the PR._

- [X] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [X] You have a descriptive commit message with a short title (first line, max 50 chars).
- [X] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [X] You have commented your code, particularly in hard-to-understand areas
- [X] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by david-allison — CHANGES_REQUESTED** [review]
Looks good!

Two variables need the `m` prefix removed as they're public variables.

`gradlew lint` to check, or wait for CI

File: AnkiDroid/src/test/java/com/ichi2/anki/dialogs/tags/TagsListTest.kt [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+import java.util.*
+import kotlin.Throws
+
+// suppressed to have a symmetry in all tests, Arrays.asList(...) should be all you need.
+@KotlinCleanup("Use kotlin functions instead of Arrays.asList")
+class TagsListTest {
+    @KotlinCleanup("non-null")
+    var mTagsList: TagsList? = null
+    @KotlinCleanup("non-null")
+    var mTagsListWithIndeterminate: TagsList? = null
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/anki/dialogs/tags/TagsListTest.kt:37]: Whoops! Submitted too early. these two variables.

**Review by david-allison — APPROVED** [review]
🚀

**Review by mikehardy — APPROVED** [review]
Nice, thanks!

**Pull request merged**